data/json/monsters.json

```diff
@@ -4493,7 +4493,7 @@
     "death_function":"NORMAL",
     "special_attacks":[["STARE", 12]],
     "description":"An eyeball the size of an easy chair, covered in rolling blue flames.  It floats through the air, spinning slowly to draw its gaze in every direction.",
-    "flags":["SEES", "WARM", "FLIES", "FIREY", "NO_BREATHE", "NOHEAD", "FAT"]
+    "flags":["SEES", "WARM", "FLIES", "FIREY", "NO_BREATHE", "NOHEAD", "FAT", "HOTAURA"]
```

Comment:
Noted. I will likely add it to the "charred nightmare" creature, and maybe create weak version of the auras to use on this creature and maybe others. Weak enough for a player to be like "damn, they thought of e'rythin'!"